cpp/cugraph_module/README.md

```diff
@@ -0,0 +1,50 @@
+## Installation setup
```

Comment:
Do we need to add these steps to the docs page under "Installing MAGE natively from the source"

Reply:
I have written the requirements, which are the same as here, but here are steps needed to take on Ubuntu, that's why I didn't write for other different systems.